packages/oscal-react-library/src/components/OSCALCatalogGroups.tsx

```diff
@@ -71,20 +78,28 @@ TabPanel.propTypes = {
  * @param {*} groups The control groupings
  * @param {*} controlLayers An decending list of groups/controls
  * @param {*} rootLayer The top most layer
- * @returns This returns the lowest found control or null if not
+ * @returns This returns the lowest found control or undefined if not
  */
-function determineControlExists(groups, controlLayers, rootLayer) {
+function determineControlExists(
+  groups: ControlGroup[] | undefined,
+  controlLayers: any,
+  rootLayer: any
```

Comment:
`controlLayers` is a list of `strings`, and `rootLayer` is a `string`. Looks like I missed updating the type for the `@param`s in JSDoc for this function. If you can, please update those as well with the types.